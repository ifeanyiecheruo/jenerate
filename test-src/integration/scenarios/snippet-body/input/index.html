<!DOCTYPE html>
<html lang="en-US">

<head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>Simple snippet test</title>
    <script src="./script.js"></script>
</head>

<body>
    <!-- This is a snippet test where x-jen-snippet elements are used to pull in snippets. -->
    <x-jen-snippet src="./body.snippet.html" />

    <!-- This snipppet will be ignored because it points at remote content -->
    <x-jen-snippet src="https://www.example.com/remote-snippet.html" /> 
    <a href="./script.js">script.js</a>
    <a href="https://www.example.com">Remote link to unknown content</a>
</body>

</html>