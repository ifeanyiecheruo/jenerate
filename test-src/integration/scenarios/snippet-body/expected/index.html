<!DOCTYPE html>
<html lang="en-US">

<head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>Simple snippet test</title>
    <script src="./script.js"></script>
</head>

<body>
    <!-- This is a snippet test where x-jen-snippet elements are used to pull in snippets. -->
    <em>This is the snippet that was included</em>
    <!-- This snipppet will be ignored because it points at remote content -->
    <a href="./script.js">script.js</a>
    <a href="https://www.example.com">Remote link to unknown content</a>
</body>

</html>